.github/dependabot.yml

```diff
@@ -10,6 +10,15 @@ updates:
     schedule:
       interval: weekly
       day: sunday
+  - package-ecosystem: docker
+    directory: /.git/logs/refs/heads
+    labels:
+      - dependencies
+      - docker
+      - Skip Changelog
+    schedule:
+      interval: weekly
+      day: sunday
```

Comment:
This doesn't seem right. Why are we adding a docker updated directive for our git reflog?

Reply:
Ah that is a mistake I'll remove that. I ran make docker-pre commit and it sometimes add this for some reason.